### DISCUSSION TITLE
基于go-cqhttp用python写了一个库，方便快速创建bot

### DISCUSSION THREAD

**Discussion by lhlyu** [Mrs4s/go-cqhttp#1924] [General]
- [链接](https://github.com/lhlyu/cqbot)

其他两个python库几乎停止了维护，所以自己就撸了一个
没有太复杂的设计，基本完成了 `event` `action` `cqcode` 相关接口
如果有什么需求欢迎提，我每天都在

**Comment by Kyomotoi** [comment]
请问你是否正在寻找：[NoneBot2](https://v2.nonebot.dev/)

**Reply by lhlyu** [reply]
用过，文档不是最新的，云里雾里的

**Reply by Kyomotoi** [reply]
文档 API 部分是根据代码注释内容自动生成的，其仅提供最基本的操作方法，如果详细到各个驱动器，你需要去看对应驱动器的文档。
云里雾里可能是你不会用，可以参考一些优秀的 [NoneBot2 实例](https://v2.nonebot.dev/store)。

**Reply by lhlyu** [reply]
实例我看过了，太复杂了，建议库里直接出个傻瓜式教程

**Reply by tom-snow** [reply]
[挺简单的啊，看官方的视频教程很快就能跑起来](https://nb2.baka.icu/docs/start/installation#通过脚手架安装推荐)，然后再从官方的[插件市场](https://nb2.baka.icu/store)找需要的插件即可

**Reply by clacknya** [reply]
nb2的文档确实很多没有写出来，术语措辞都很突然。我是读了大半天源码才明白讲的啥。不过了解之后还是挺好用的。

**Reply by lhlyu** [reply]
> nb2的文档确实很多没有写出来，术语措辞都很突然。我是读了大半天源码才明白讲的啥。不过了解之后还是挺好用的。

同感

**Reply by Well2333** [reply]
> nb2的文档确实很多没有写出来，术语措辞都很突然。我是读了大半天源码才明白讲的啥。不过了解之后还是挺好用的。

新文档在路上了
